backend/src/routes/api/k8s/index.ts

```diff
@@ -43,10 +43,13 @@ module.exports = async (fastify: KubeFastifyInstance) => {
       };
 
       let promise: Promise<unknown>;
-      if (req.headers.accept === 'text/plain') {
-        promise = passThroughText(fastify, req, passThroughData);
-      } else {
-        promise = passThroughResource(fastify, req, passThroughData);
+
+      switch (req.types(['json', 'text', 'html'])) {
```

Comment:
Yes, it goes to default.

Reply:
So it will not parse the data.

Reply:
@DaoDaoNoCode what is the value behind `req.types([...])`?

Reply:
if we don't pass anything to `req.types`, the value would be a `string[]` which is a list of all the types in accept header. If we pass the types into `req.types`, it will go over the accept header and return a string, which is the first match type. If nothing matches, it will return a boolean `false`.